interpreter/src/main/java/org/qbicc/interpreter/impl/VmImpl.java

```diff
@@ -524,6 +524,10 @@ public void initialize() {
                 VmClassImpl clazz = cl.loadClass(def.getInternalName());
                 return clazz;
             });
+            reflectClass.registerInvokable("getClassAccessFlags", (thread, target, args) -> {
+                VmClassImpl cls = (VmClassImpl)args.get(0);
+                return cls.getTypeDefinition().getModifiers();
```

Comment:
This needs to be masked with `0x1FFF` as per the spec in `jdk.internal.reflect.Reflection#getClassAccessFlags` in order to prevent our extra flags from leaking out.

Reply:
fixed.